README.md

```diff
@@ -31,6 +33,10 @@ https://github.com/doomers/Remote-Desktop/issues
 Patches are encouraged, and may be submitted by forking this project and
 submitting a pull request through GitHub. Please see CONTRIBUTING.md for more details.
 
+Contributors
+------------
+- magrandera
+
```

Comment:
can you add contributors link instead of only your name
contributor link is:
https://github.com/doomers/Remote-Desktop/graphs/contributors